spec/unit/git/cache_spec.rb

```diff
@@ -3,6 +3,16 @@
 
 describe R10K::Git::Cache do
 
+  describe 'the default cache_root' do
+    it 'is in the right location in linux', unless: described_class.windows? do
+      expect(described_class.defaults[:cache_root]).to match(/\.r10k\/git/)
+    end
+
+    it 'is in the right location for windows', if: described_class.windows? do
+      expect(described_class.defaults[:cache_root]).to match(/\.{0}r10k\/git/)
```

Comment:
what is this regex doing? does that mean "zero periods before 'r10k/git'"?  it doesn't do what I'd expect locally:
```rb
irb(main):003:0> '/Users/justin/.r10k/git' =~ /\.r10k\/git/
=> 14

irb(main):004:0> '/Users/justin/.r10k/git' =~ /\.{0}r10k\/git/
=> 15
```
If that's what you're going for I've usually seen "not a period" expressed as `[^.]`
```rb
irb(main):005:0> '/Users/justin/.r10k/git' =~ /[^.]r10k\/git/
=> nil

irb(main):006:0> '/Users/justin/r10k/git' =~ /[^.]r10k\/git/
=> 13
```

Reply:
I swear I ran this through rubular.com, but now I can't repro the results. I'll update it to the bracket notation.